### PR TITLE
JBDS-3495 add Docker plugins (not feature) to...
... JBT site to avoid p2.inf config instruction

### PR DESCRIPTION
**Pull request by nickboldt**

### CODE REVIEW

File: aggregate/site/category.xml [hunk 1]
```diff
@@ -284,10 +284,20 @@
     <category name="CoreTools" />
     <category name="MavenTools"/>
   </feature>
-  <feature id="org.eclipse.linuxtools.docker.feature">
+  <!-- JBDS-3495 add plugins instead of feature to avoid p2.inf instruction which forces JBDS to include Docker nightly update site in disabled state -->
```
Comment (maxandersen) [inline aggregate/site/category.xml]: why is this change necessary when we removed the p2.inf instructions ?

Comment (nickboldt) [inline aggregate/site/category.xml]: Because removing them from the update site doesn't remove them from the feature itself. My previous solution was incomplete and full o' fail. This one actually works.
